packages/rainbowkit/src/wallets/walletConnectors/compassWallet/compassWallet.ts

```diff
@@ -0,0 +1,15 @@
+import { Wallet } from '../../Wallet';
+import { getInjectedConnector } from '../../getInjectedConnector';
+
+export const compassWallet = (): Wallet => ({
+  id: 'compass',
+  name: 'Compass Wallet',
+  installed: !!window.compassEvm,
+  iconUrl: async () => (await import('./compassWallet.svg')).default,
+  iconBackground: '#fff',
+  downloadUrls: {
+    chrome:
+      'https://chromewebstore.google.com/detail/compass-canary-build/lifffofignanainchjlljapjekjjlaeb?hl=en-GB&authuser=0',
```

Comment:
I think we need to wait until this wallet is released in production

Reply:
This wallet is related to Sei EVM development, and as that is something in continuous development, this wallet is also in development. That's why a canary version, so dApp can easily integrate and test out the transactions.

May I know (if it's okay), why do we need to wait till production?

Reply:
I was also thinking, that I will need to raise one more PR once Sei EVM goes live, as then this wallet will be in production.